packages/format-library/src/link/inline.js

```diff
@@ -119,8 +119,13 @@ function InlineLinkUI( {
 		} );
 
 		if ( isCollapsed( value ) && ! isActive ) {
+			const newText =
+				nextValue.hasOwnProperty( 'title' ) && nextValue.title !== ''
```

Comment:
Do you see any benefit of this implementation vs...

```suggestion
				nextValue.title
```

?

Or even simplified further:

```js
const newText = nextValue.title || newUrl;
```

Reply:
That seems to work just as well. Looks a lot cleaner! I noticed the title property will not always exist inside nextValue and my PHP mind was thinking this would throw an error, but seems like in JS nextValue.jibberish will just return undefined, which is falsy. So yeah, this will work just the same. Will update the PR.

Reply:
Yeah, the original implementation could make sense if you needed to differentiate `undefined` or `''` (empty string), but as you mention, it should be fine enough to simply use falsyness as an indicator.